pkg/skaffold/kubernetes/manifest/manifests.go

```diff
@@ -17,16 +17,33 @@ limitations under the License.
 package manifest
 
 import (
+	"bufio"
 	"bytes"
 	"io"
 	"regexp"
 	"strings"
+
+	k8syaml "k8s.io/apimachinery/pkg/util/yaml"
 )
 
 // ManifestList is a list of yaml manifests.
 //nolint:golint
 type ManifestList [][]byte
 
+// Load uses the Kubernetes `apimachinery` to split YAML content into a set of YAML documents.
+func Load(in io.Reader) ManifestList {
```

Comment:
how about `func Load(in io.Reader) (ManifestList, error)` and we can handle the error?

Reply:
👍